Document/0x06g-Testing-Network-Communication.md

```diff
@@ -204,7 +204,7 @@ In general it can be summarized:
 
 #### Overview
 
-Certificate pinning is the process of associating the mobile app with a particular X.509 certificate of a server, instead of accepting any certificate signed by a trusted certificate authority. A mobile app that stores ("pins") the server certificate or public key will subsequently only establish connections to the known server. By removing trust in external certificate authorities, the attack surface is reduced (after all, there are many known cases where certificate authorities have been compromised or tricked into issuing certificates to impostors).
+Certificate pinning is the process of associating the mobile app with a particular X.509 certificate of a server, instead of accepting any certificate signed by a trusted certificate authority. A mobile app that stores ("pins") the server certificate or public key will subsequently only establish connections to the known server. By removing trust in external certificate authorities, the attack surface is reduced. After all, there are many known cases where certificate authorities have been compromised or tricked into issuing certificates to impostors. A detailed timeline of CA breaches and failures can be found on [sslmate.com](https://sslmate.com/certspotter/failures "Timeline of PKI Security Failures").
```

Comment:
> A mobile app that stores ("pins") the server certificate or public key will subsequently only establish connections to the known server.

To

> A mobile app that stores the server certificate or public key will subsequently only establish connections to the known server, thereby "pinning" the server.

Reply:
> certificate authorities

to

> certificate authorities (CAs)

So that the CA referenced later has a meaning (unless it's already explained earlier?)

Reply:
> on sslmate

or

> at sslmate.com

https://english.stackexchange.com/questions/102806/can-be-found-in-can-be-found-on-prepositions

Reply:
agree. thanks